encord/objects/common.py

```diff
@@ -43,14 +42,24 @@ class Attribute(ABC):
     that are part of an :class:`encord.objects.ontology_object.Object`.
     """
 
+    def __init__(self, uid: NestedID, feature_node_hash: str, name: str, required: bool, dynamic: bool):
+        self.uid = uid
+        self.feature_node_hash = feature_node_hash
+        self.name = name
+        self.required = required
+        self.dynamic = dynamic
+
+    @property
+    def options(self) -> Sequence[Option]:
+        return []
```

Comment:
I'm curious, why do you think this is better than explicitly returning `None` for Text attributes?